src/main/java/blackjack/domain/player/Player.java

```diff
@@ -19,4 +25,20 @@ public void compareWithDealer(Dealer dealer) {
     public boolean canDrawOneMore(){
         return getScore() <= LIMIT_SCORE;
     }
+
+    public boolean isLose(Result result) {
+        return result == Result.LOSE;
+    }
+
+    public float profit(Result result) {
+        if (isBlackjack()) {
+            return betMoney.getBlackjackBetMoney();
+        }
+
+        if (isLose(result)) {
+            return -betMoney.getBetMoney();
```

Comment:
```suggestion
            return -1 * betMoney.getBetMoney();
```
가독성을 위해 이게 조금 더 나을 것 같아요 :)